pkg/minikube/cluster/cluster.go

```diff
@@ -104,10 +105,10 @@ func StartHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 func StopHost(api libmachine.API) error {
 	host, err := api.Load(constants.MachineName)
 	if err != nil {
-		return err
+		return errors.New(err.Error())
```

Comment:
Can we use `errors.Wrap` with added context in all places with `errors.New`? Otherwise we lose the type of the original error (not necessarily important, but potentially useful).